sponsorship_compassion/models/contracts.py

```diff
@@ -623,53 +623,8 @@ def suspend_contract(self):
                 message_type="comment",
             )
 
-        # Change invoices if config tells to do so.
```

Comment:
In fact you can completely remove this method and the code that calls it. We no longer need anything from that. You can also remove the code that inherits `def suspend_funds` in `sponsorship_compassion/models/project_compassion.py` at line 48.

Reply:
You can also remove field `suspended_amount` and its compute method.